wordpress-mobile/WordPress-Android
# Feature/single gif picker support

This PR adds single selection support to a tenor GIF picker in #12161

Multi-selection functionality is also preserved since it's used in the media browser, and can potentially be used later in editor.

I used a stock media picker for a reference but decided not to implement blocking progress dialog used there.

![gif](https://user-images.githubusercontent.com/728822/85214149-5d0fe380-b31c-11ea-8f20-6d744c33ddf6.gif)

To test (same as in #12161)

Using zalpha flavor:

- Open a post in Gutenberg
- Add an image
- Notice that there is an "Add from Tenor" option
- Click on it
- Search for a GIF
- Add the GIF to the post
- Check that it's correctly uploaded and the post could be saved

In addition:
- Navigate to media browser.
- Tap on the plus icon in the toolbar and select "Chose from Tenor"
- Select a single gif and add it to a media library.
- Repeat the steps above but this time select multiple GIFs and add them to a library.
- Confirm that the functionality works as expected.

To test:

PR submission checklist:
